src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -66,15 +78,15 @@ const ProgressTracker = ({
       if (value) {
         return value;
       }
-      return showDefaultLabels ? defaultValue : undefined;
+      return showDefaultLabels ? defaultValue : null;
```

Comment:
Does it need to be null? It doesn't matter just wondering why the change was made